examples/simple-platform/platform-structure.md

```diff
@@ -0,0 +1,45 @@
+# Platform structure
+
+Our platform has two server nodes. One runs the `platform-deploy` hypervisor 
+and the other runs the `web-auth-endpoint` hypervisor. We also have two 
+browser nodes - one for the accounts app and another for the template app. 
+These nodes run the React apps in the browser.
+
+The `platform-deploy` node hosts the accounts and template chains, acting as 
+the server for the two apps. It is the blockmaster and thus is responsible for 
+adding new blocks to the application chains. Note that there is no requirement 
+for the accounts and template chains to be hosted on the same node. They could 
+be hosted on separate nodes and moreover each app's chains could be hosted on 
+multiple nodes for redundancy. (FIXME: Should we mention that this has not 
+been implemented yet?) 
+
+The `web-auth-endpoint` node hosts the accounts app chains, which includes a 
+GitHub chain. It also runs an Express instance that fields the http requests 
+coming from GitHub during an OAuth loop. 
+
+These two server nodes are peers. During the GitHub OAuth loop, OAuth requests 
+from  GitHub are received by the Express instance. An action is dispatched to 
+the GitHub chain on the `web-auth-endpoint` hypervisor which then passes it 
+along to the `platform-deploy` hypervisor. The `app-account` control chain 
```

Comment:
While the idea of passing the action along to the other hypervisor is correct it ignores several steps and makes it difficult to understand the underlying behaviour. I think mentioning the chain joining and gossiping that makes this action passing possible would help clarify. Alternatively, finding different wording that is more vague but still descriptive enough to indicate gossip/joining would work too.

Reply:
I think there is some confusion between cli and hypervisor. The express instance uses the web-auth-endpoint cli to dispatch an action that is distributed to, and processed deterministically by all nodes on the network. The chains and inter-node connections are managed by the hypervisor.